cmd/commands/destroy_cluster.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	//cephRookApiV1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	"fmt"
+	"github.com/rook/kubectl-rook-ceph/pkg/crds"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/spf13/cobra"
+	"log"
+	//https://github.com/rook/rook/tree/master/pkg/apis/ceph.rook.io/v1
+)
+
+// DestroyClusterCmd represents the command for destroy cluster
+var DestroyClusterCmd = &cobra.Command{
+	Use:   "destroy-cluster",
```

Comment:
```suggestion
	Use:   "destroy-cluster",
```
how about naming it `clean-cluster` or `wipe-cluster`? destroy sounds strange to me for command IMO

Reply:
We need to be very clear that this command is destructive and would cause all data to be lost, so I though a scary name like `destroy-cluster` makes sense. `clean` and `wipe` don't sound scary enough to me.